concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-876	876	Markt	Markt	market	633	MARKET
+Steinthal-1875-1565-877	877	Preis	Preis	price	657	PRICE
+Steinthal-1875-1565-878	878	Geld	Geld	money	1559	MONEY
+Steinthal-1875-1565-879	879	theuer	theuer	expensive	1426	EXPENSIVE
+Steinthal-1875-1565-880	880	billig	billig	cheap	1887	CHEAP
+Steinthal-1875-1565-881	881	Lohn	Lohn	wage	1122	WAGES
+Steinthal-1875-1565-882	882	Pfand	Pfand	deposit		
+Steinthal-1875-1565-883	883	bezahlen	bezahlen	to pay	718	PAY
+Steinthal-1875-1565-884	884	Verdienst	Verdienst	earnings		
+Steinthal-1875-1565-885	885	Verlust	Verlust	loss		
+Steinthal-1875-1565-886	886	schenken	schenken	to gift		
+Steinthal-1875-1565-887	887	borgen	borgen	to borrow 	1823	BORROW
+Steinthal-1875-1565-888	888	miethen	miethen	to rent	3213	HIRE OR RENT
+Steinthal-1875-1565-889	889	leihen	leihen	to lend	582	LEND
+Steinthal-1875-1565-890	890	tauschen	tauschen	to exchange	2509	EXCHANGE
+Steinthal-1875-1565-891	891	Ich bin ihm schuldig.	Ich bin ihm schuldig.	I owe him.	369	OWE
+Steinthal-1875-1565-892	892	Ich bekomme zum Geschenk.	Ich bekomme zum Geschenk.	I receive something as a present.		
+Steinthal-1875-1565-893	893	wiedergeben	wiedergeben	to return something		
+Steinthal-1875-1565-894	894	reich	reich	rich	712	RICH
+Steinthal-1875-1565-895	895	arm	arm	poor	1674	POOR
+Steinthal-1875-1565-896	896	zählen	zählen	to count	1420	COUNT
+Steinthal-1875-1565-897	897	Häuptling	Häuptling	chieftain	301	CHIEFTAIN
+Steinthal-1875-1565-898	898	König, Königin, deren Sohn	König	king	1508	KING
+Steinthal-1875-1565-899	899	König, Königin, deren Sohn	Königin	queen	1306	QUEEN
+Steinthal-1875-1565-900	900	König, Königin, deren Sohn	Prinz	prince	2554	PRINCE (RULER)
+Steinthal-1875-1565-901	901	freier Mann	freier Mann	free man	384	FREEMAN
+Steinthal-1875-1565-902	902	berathende Versammlung	berathende Versammlung	gathering		
+Steinthal-1875-1565-903	903	Sklave	Sklave	male slave	812	SLAVE
+Steinthal-1875-1565-904	904	Sklavin	Sklavin	female slave	812	SLAVE
+Steinthal-1875-1565-905	905	befehlen	befehlen	to order	1128	ORDER
+Steinthal-1875-1565-906	906	gehorchen	gehorchen	to obey	1635	OBEY
+Steinthal-1875-1565-907	907	gehorsam	gehorsam	obedient		
+Steinthal-1875-1565-908	908	Priester	Priester	priest	852	PRIEST
+Steinthal-1875-1565-909	909	Zauberer (Arzt)	Zauberer	sorcerer	823	SORCERER
+Steinthal-1875-1565-910	910	Zauberer (Arzt)	Arzt	doctor	597	PHYSICIAN
+Steinthal-1875-1565-911	911	Arznei	Arznei	medicine	1372	MEDICINE
+Steinthal-1875-1565-912	912	Opfer	Opfer	sacrifice	1103	SACRIFICE
+Steinthal-1875-1565-913	913	beten (Ceremonien beim Beten)	beten	to pray	24	PRAY
+Steinthal-1875-1565-914	914	beten (Ceremonien beim Beten)	Ceremonien beim Beten	ceremonies while praying		
+Steinthal-1875-1565-915	915	fasten	fasten	to fast	303	ABSTAIN FROM FOOD
+Steinthal-1875-1565-916	916	schwören	schwören	to swear	1939	SWEAR
+Steinthal-1875-1565-917	917	Geister beschwören	Geister beschwören	to conjure spirits		
+Steinthal-1875-1565-918	918	fluchen	fluchen	to curse	8	CURSE
```

Comment:
I think our course (8) is the 'to (magically) curse someone/somebody', is this here the general 'to curse'?

Reply:
I don't know. I assumed it would be the ('ver)fluchen, Fluch aussprechen' meaning instead of 'using swear words' because it is surrounded by Geister beschwören and segnen which have this magic connotation as well. But we can unmap it to be sure.